<?xml version="1.0"?>
<launch>
  <arg name="pub_rate"         default="1"/>
  <arg name="resume_data_dir"  default="False"/>
  <arg name="candidates_path"  default="$(find bayesopt_ros_bridge)/examples/data/candidates.csv"/>
  <arg name="data_save_dir"    default="$(find bayesopt_ros_bridge)/examples/data/result"/>
  <arg name="config_file"      default="$(find bayesopt_ros_bridge)/config/bridge_config_physbo.yaml"/>

  <node name="physbo_ros_bridge" pkg="bayesopt_ros_bridge" type="physbo_ros_bridge_node.py" output="screen">
    <param name="pub_rate"         value="$(arg pub_rate)"/>
    <param name="resume_data_dir"  value="$(arg resume_data_dir)"/>
    <param name="candidates_path"  value="$(arg candidates_path)"/>
    <param name="data_save_dir"    value="$(arg data_save_dir)"/>
    <rosparam command="load" file="$(arg config_file)" />
  </node>
</launch>
